### PR TITLE
Documentation Overhaul

### PR DESCRIPTION
**Pull request by mikereinhart**
Adding a node that install.sh will run automated tests prior to installing the app on your device

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Installation requirements differ based on installation method:
 - **Installing Release Files**: Release files can be installed using Linux and Mac. Windows is not supported.
 - **Installing from Source**: The application can be built from source and installed on Linux machines with the Nix Package Manager.
 
-This applications are built against Ledger Nano S firmware 1.6.0 and Ledger Nano X firmware XXX. Please use [Ledger Live](https://www.ledger.com/ledger-live) to manage your Ledger device's firmware.
+This applications are built against Ledger Nano S firmware 1.6.0 and Ledger Nano X firmware 1.2.4-1. Please use [Ledger Live](https://www.ledger.com/ledger-live) to manage your Ledger device's firmware.
```
Comment (jmininger) [inline README.md:19]: These

Comment (mikereinhart) [reply README.md]: Good catch! Fixed

File: README.md [hunk 2]
```diff
@@ -257,47 +260,43 @@ CKB> account import --ledger 0x69c46b6dd072a2693378ef4f5f35dcd82f826dc1fdcc89125
...
+| `Public Key` 	| `ckb1qyqxxtzygxvjwhgqklqlkedlqqwhp0rqjkvsqltkvh` 	|
 
-``` text
-Address:
-ckt1qyqxxtzygxvjwhgqklqlkedlqqwhp0rqjkvsa64fqt
-```
+**Verifying the output address printed by `ckb-cli` matches the one shown on Ledger prompt is highly recommended. Please read [Ledger's documentation](https://support.ledger.com/hc/en-us/articles/360006433934) on the subject.**
 
-After accepting the prompt on the Ledger the output on `ckb-cli` should look like:
+*Note: If you've changed the Ledger's configuration to show testnet address, the `Address` prompt will instead show the testnet address. This setting persists between power cycles.*
```
Comment (jmininger) [inline README.md:297]: *"if youve changed the nervos-app's configuration" maybe?

Comment (mikereinhart) [reply README.md]: Pushed a fix, how's this?

**Pull request merged**